app/about/teammembers/[id]/page.js

```diff
@@ -0,0 +1,15 @@
+// (about/teammembers/[id]/page.js)
+import teamMembers from "../../../../public/images/members";
+import Frame from "../../../../components/pages/About/widgets/Frame";
+export default function Page({ params }) {
+  console.log("id", params.id);
```

Comment:
Delete or comment out 'console.logs()' if they are unnecessary.

Reply:
Ok, I'll delete it.